model/result/ResultService.php

```diff
@@ -141,8 +129,12 @@ protected function handleReplaceRequest(LisOutcomeRequest $request, DeliveryExec
         /** @noinspection PhpUnhandledExceptionInspection */
         $eventManager->trigger(new LisScoreReceivedEvent($deliveryExecution->getIdentifier()));
 
-        $this->logInfo(sprintf("Score '%s' added for delivery execution '%s'",
-            $operationRequest->getScore(), $operationRequest->getSourcedId())
+        $this->logInfo(
+            sprintf(
+            "Score '%s' added for delivery execution '%s'",
+            $operationRequest->getScore(),
+            $operationRequest->getSourcedId()
+        )
```

Comment:
```suggestion
        $this->logInfo(
            sprintf(
                "Score '%s' added for delivery execution '%s'",
                $operationRequest->getScore(),
                $operationRequest->getSourcedId()
        )
```
Small indentation issue